doc/themes/the-things-stack/assets/css/shortcodes/messages.scss

```diff
@@ -0,0 +1,61 @@
+
+.api-messages-selector-container{
+  margin-bottom: 1rem;
+  padding: 1rem;
+  border-radius: 0.25rem;
+}
+
+.api-messages-selector{
+  padding-bottom: 1rem;
+}
+
+.api-messages-container {
+  margin-bottom: 1rem;
+  padding: 0.5rem 1rem;
+  border-radius: 0.25rem;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+}
+
+.api-message-container {
+  border: 1px solid #e5e5e5;
+  border-radius: 0.25rem;
+  padding: 1rem;
+  margin-bottom: 2rem;
+  width: 100%;
+}
+
+.api-message-description{
+  margin-bottom: 1rem;
+}
+
+.api-message-table .table{
+  width: 100%;
+}
+
+.api-message-schema {
+  margin-bottom: 1rem;
+  padding-top: 1rem;
+}
+
+.api-message-example {
+  margin-bottom: 1rem;
+  padding-top: 1rem;
+}
+
+.api-message-enum-list {
+  padding: 1rem;
+}
+
+.api-message-float-button {
+	position:fixed;
+	width:20px;
+	height:20px;
+	bottom:40px;
+	right:40px;
+	border-radius:50px;
+	text-align:center;
+	box-shadow: 2px 2px 3px #999;
+}
```

Comment:
This still doesn't look good on GitHub. Not only indentation, also spacing. Can you use a formatter please